chat-server/src/utils.ts

```diff
@@ -23,15 +23,37 @@ export const getRequestId = (req: ExpressRequest) => {
   }
 };
 
-export const sendErrorResponse = (
-  res: ExpressResponse,
-  httpStatus: number,
-  errorMessage: string,
-  errorDetails?: string
-) => {
-  logger.error(
-    stripIndent`Responding with ${httpStatus} status and error message: ${errorMessage}.
-    ${errorDetails ? `Error details: ${errorDetails}` : ""}`
-  );
+export interface LogRequestParams {
+  reqId: string;
+  message: string;
+  type?: "info" | "error";
+}
+
+export const logRequest = ({ reqId, message, type }: LogRequestParams) => {
+  type = type || "info";
```

Comment:
I think this can be done above
```{ reqId, message, type = "info" }```
In general it's a bad practice to reassign argument variables.